sf-fx-runtime-java-logger/src/test/java/com/salesforce/functions/jvm/runtime/logger/DefaultLoggingFormatterTest.java

```diff
@@ -66,6 +70,19 @@ public void testEmptyMDC() {
         result,
         is(
             equalTo(
-                "00:00:00.000 TRACE [INVOCATION null] c.s.f.jvm.runtime.logger.EmptyMDC - This is a message!\n")));
+                String.format(
+                    "\"UTC\"=\"00:00:00.000\" \"level\"=\"TRACE\" \"loggerName\"=\"c.s.f.jvm.runtime.logger.EmptyMDC\" \"message\"=\"This is a message!\" \"invocationId\"=\"null\" \n"))));
+  }
+
+  @Test
+  public void testQuotations() {
+    String result = formatter.format("blank", Level.INFO, "Checking Quotations \"test\"");
+
+    assertThat(
+        result,
+        is(
+            equalTo(
+                String.format(
+                    "\"UTC\"=\"00:00:00.000\" \"level\"=\"INFO\" \"loggerName\"=\"blank\" \"message\"=\"Checking Quotations \"test\"\" \"invocationId\"=\"e3a4ae2b-fefb-4277-89d0-7068e7e39b99\" \n"))));
```

Comment:
This is incorrect. The expected result would be:
```
"\"UTC\"=\"00:00:00.000\" \"level\"=\"INFO\" \"loggerName\"=\"blank\" \"message\"=\"Checking Quotations \\\"test\\\"\" \"invocationId\"=\"e3a4ae2b-fefb-4277-89d0-7068e7e39b99\" \n"))));
```
Since this is a String literal, all escapes are for the literal, not the actual string. That's why we'd need to double escape the backslashes and quotes for around `"test"`. 

When this is fixed, this will show that quoting is not implemented yet.